### PR TITLE
Approve url used for Approve/Decline webpage concatenation

### PR DESCRIPTION
**Pull request by Kaibanda**

### CODE REVIEW

File: booking-app/app/approve/page.tsx [hunk 1]
```diff
@@ -59,12 +115,11 @@ const ApprovePageContent: React.FC = () => {
   );
 };
 
-const ApprovePage: React.FC = () => (
-  <DatabaseProvider>
-    <Suspense fallback={<div>Loading...</div>}>
-      <ApprovePageContent />
-    </Suspense>
-  </DatabaseProvider>
+const ApproveDeclinePage: React.FC = () => (
```
Comment (rlho) [inline booking-app/app/approve/page.tsx:118]: `<DatabaseProvider>` is necessary.
Use the latest code from the main branch.

**Comment by rlho** [comment]
@Kaibanda 
Also, with this UI, it appears that I can't decline.
Screenshot 2024-10-10 at 3 20 04 PM

**Comment by nopivnick** [comment]
> Also, with this UI, it appears that I can't decline.

I believe we said the Decline button should not be click-able until something has been entered in the "Reason for Declining" text field.

**Comment by nopivnick** [comment]
@Kaibanda an alert with a message sounds good.

**Comment by Kaibanda** [comment]
The functionality for the alert message was already implemented, so I changed that back to how it was previously.

**Comment by rlho** [comment]
@Kaibanda Please resolve conflicts!

**Pull request closed without merging**